### ISSUE TITLE
Semantic Cite Update

### ISSUE THREAD

**Issue by akshamoor** [SemanticMediaWiki/SemanticCite#124]
Good Day:

Semantic Cite is an excellent extension. But I have FOMO for the latest Semantic MediaWiki.

I have failed to install Semantic Cite under: MediaWiki higher than 1.35.x and Semantic MediaWiki higher than 3.2.3.

I have attempted without success:
composer update --with-all-dependencies
composer composer update --no-dev

I have attempted this on a Mac, on Windows 10 and 11 using XAMPP and under Windows 11 > WSL2 > Ubuntu 20.04. Retrying is an annual hobby. 

My present versions:
Windows 11
Microsoft WSL2
Ubuntu 20.04. 
PHP: 7.4.3
MariaDB: 10.3.39
MediaWiki: 1.35.10
Semantic MediaWiki: 3.2.3
Semantic Cite: 2.1.1

My wiki is local only; no external access. 

Here is an error message. Please excuse my use of PRE tags. 

<pre>Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires mediawiki/semantic-cite ~2.1 -> satisfiable by mediawiki/semantic-cite[2.1.0, 2.1.1].
    - mediawiki/semantic-cite[2.1.0, ..., 2.1.1] require mediawiki/semantic-media-wiki ~3.0 -> found mediawiki/semantic-media-wiki[3.0.0, ..., 3.2.3] but it conflicts with your root composer.json require (~4.1).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.</pre>

Thank you
Clarence Pettis

**Comment by z929669** [comment]
Senantic Cite 3.0.0 installs and performs for me without issue under Semantic Mediawiki 4.1.3, resolving all of my previous issues with v2.1 under Semantic Mediawiki 4.0. Also, #107 is closed now and consistent with my current results (under SMW ~4.0).

Thanks all.

**Comment by krabina** [comment]
Did you try running update.php? That should check for all tables needed.

**Comment by z929669** [comment]
Yes, I did run update.php when I updated PHP from 8.0-->8.1 and MW from 1.39.5-->1.39.7. All was kosher then, because the tables were created properly. However, some process nuked the tables I mentioned, and it would've had to be a SMW maintenance script I have in cron. I can think of nothing else that would cause it (since I didn't change anything). I copied a snapshot of my server and switched the floating IP over once all was updated is all.
